### PR TITLE
Cleanup: remove prometheus dependencies for volume

### PR DESCRIPTION
**Pull request by gavinfish**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

remove prometheus dependencies for volume

**Which issue(s) this PR fixes**:

Xref #89267

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
none
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/volume/util/metrics.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ var storageOperationMetric = metrics.NewHistogramVec(
 	[]string{"volume_plugin", "operation_name", "status"},
 )
 
-var storageOperationStatusMetric = metrics.NewCounterVec(
+var StorageOperationStatusMetric = metrics.NewCounterVec(
```
Comment (gavinfish) [inline pkg/volume/util/metrics.go:53]: I have not found any criterion to decide the package visibility of such metric variables. But this should be OK according to similar ones in kubelet and scheduler.

https://github.com/kubernetes/kubernetes/blob/bff6452893e58e269290fe321c21285244ecb95f/pkg/kubelet/metrics/metrics.go#L86-L94

https://github.com/kubernetes/kubernetes/blob/bff6452893e58e269290fe321c21285244ecb95f/pkg/scheduler/metrics/metrics.go#L98-L107

**Comment by gnufied** [comment]
@gavinfish can you perhaps rebase the PR. I am sorry that it we did not review this, but I will take a look.

**Comment by gavinfish** [comment]
@gnufied rebased.

**Comment by lavalamp** [comment]
/approve

for /hack owners. Others will have to review & lgtm.

**Comment by dims** [comment]
Is this PR still needed, please rebase if so (or we can close it?)

**Comment by dims** [comment]
This PR is older than 4 weeks and needs as rebase.

Please rebase the PR against latest master and reopen if still needed.

/close

**Pull request closed without merging**